package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@trishnakalita660 can you delete this file? I think it was accidentally created.